setup.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+
+
+from setuptools import setup
```

Comment:
setuptools should be added to the [dependencies](https://github.com/SciTools-incubator/iris-ugrid/blob/62c8e27b1e3f46b4042605225ae5e00be27b630f/requirements/conda_requirements.txt)

Reply:
(I will shortly be aligning the dependency structure with the structure already used in Iris and elsewhere - see #31 - so no need to be concerned with the simple structure at this point)

Reply:
Well spotted - thanks @trexfeathers. Done this now.